src/internal/observable/dom/AjaxObservable.ts

```diff
@@ -21,34 +20,42 @@ export interface AjaxRequest {
   responseType?: string;
 }
 
+// Declare older APIs we'll check for on global scope.
+declare const XDomainRequest: any;
+declare const ActiveXObject: any;
+
+function isFormData(body: any): body is FormData {
+  return typeof FormData !== 'undefined' && body instanceof FormData;
+}
+
 function getCORSRequest(): XMLHttpRequest {
-  if (root.XMLHttpRequest) {
-    return new root.XMLHttpRequest();
-  } else if (!!root.XDomainRequest) {
-    return new root.XDomainRequest();
+  if (XMLHttpRequest) {
```

Comment:
With these no longer being property accesses, I think tests like `if (typeof XMLHttpRequest === 'undefined`) {` will be necessary to avoid errors being thrown.

Reply:
added